app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -27,6 +25,15 @@ def index
       respond(collection: collection)
     end
 
+    api :PUT, "/organizations/:organization_id/upstream_subscriptions",
```

Comment:
hrmmm, using PUT here seems weird, what is the update quantity going to look like?  We really should have planned out all these apis from the start :)

Reply:
Yeah, should be a DELETE, huh :)